package.json

```diff
@@ -13,6 +13,7 @@
     "build": "run-p build:*",
     "build:cli": "ncc build -o build --minify src/bin/entry-point/cli.ts",
     "build:action": "ncc build -o dist --minify src/bin/entry-point/github-action.ts",
+    "dev:build": "tsc -p tsconfig.json && tsc-alias -p tsconfig.json",
```

Comment:
I think simply `tsc && tsc-alias` should work since we are using the default `tsconfig.json`

Reply:
yeah I can confirm it works also with simple `tsc && tsc-alias`